[2022-10-03 17:14:43] console.CRITICAL: Error thrown while running command ""server:run"". Message: "There are no commands defined in the "server" namespace.  You may be looking for a command provided by the "WebServerBundle" which is currently not installed. Try running "composer require symfony/web-server-bundle --dev"." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): There are no commands defined in the \"server\" namespace.\n\nYou may be looking for a command provided by the \"WebServerBundle\" which is currently not installed. Try running \"composer require symfony/web-server-bundle --dev\". at C:\\wamp64\\www\\atypik\\vendor\\symfony\\framework-bundle\\EventListener\\SuggestMissingPackageSubscriber.php:74)","command":"\"server:run\"","message":"There are no commands defined in the \"server\" namespace.\n\nYou may be looking for a command provided by the \"WebServerBundle\" which is currently not installed. Try running \"composer require symfony/web-server-bundle --dev\"."} []
[2022-10-03 17:17:51] console.CRITICAL: Error thrown while running command ""dotenv:dump"". Message: "There are no commands defined in the "dotenv" namespace." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\NamespaceNotFoundException(code: 0): There are no commands defined in the \"dotenv\" namespace. at C:\\wamp64\\www\\atypik\\vendor\\symfony\\console\\Application.php:604)","command":"\"dotenv:dump\"","message":"There are no commands defined in the \"dotenv\" namespace."} []
[2022-10-03 17:58:10] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 17:58:10] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 17:58:10] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:10] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:11] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 17:58:11] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 17:58:37] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 17:58:37] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 17:58:37] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:37] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:37] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 17:58:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 17:58:40] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 17:58:40] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 17:58:40] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:40] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:40] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 17:58:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 17:58:49] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 17:58:49] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 17:58:49] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:49] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:49] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 17:58:49] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 17:58:51] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 17:58:51] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 17:58:51] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:51] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 17:58:51] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 17:58:51] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:08:08] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:08:08] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:08:08] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:08] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:09] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:08:09] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:08:12] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:08:13] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:08:13] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:13] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:13] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:08:13] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:08:42] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:08:42] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:08:42] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:42] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:08:42] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:08:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:09:14] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:09:14] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:09:14] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:09:14] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:09:14] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:09:14] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:10:34] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:10:34] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:10:34] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:10:34] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:10:34] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:10:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-10-03 18:11:45] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2022-10-03 18:11:45] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-10-03 18:11:45] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:11:45] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-10-03 18:11:45] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI)" at C:\wamp64\www\atypik\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 1045): An exception occurred in the driver: SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\API\\MySQL\\ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:28, PDOException(code: 1045): SQLSTATE[HY000] [1045] Accès refusé pour l'utilisateur: 'snryljie_Yacein'@'@localhost' (mot de passe: OUI) at C:\\wamp64\\www\\atypik\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\MySQL\\Driver.php:28)"} []
[2022-10-03 18:11:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2022-11-27 17:36:20] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"https://f2i-dev23-yk-yb-mz.fr/public/","method":"GET"} []
[2022-11-27 17:36:20] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2022-11-27 17:36:20] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-11-27 17:36:20] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2022-11-27 17:36:20] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[HY000] [2002] Can't connect to MySQL server on '127.0.0.1' (115)" at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 101 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 2002): An exception occurred in the driver: SQLSTATE[HY000] [2002] Can't connect to MySQL server on '127.0.0.1' (115) at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:101, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 2002): SQLSTATE[HY000] [2002] Can't connect to MySQL server on '127.0.0.1' (115) at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28, PDOException(code: 2002): SQLSTATE[HY000] [2002] Can't connect to MySQL server on '127.0.0.1' (115) at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:28)"} []
[2022-11-27 17:36:20] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2023-02-12 17:13:47] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"https://f2i-dev23-yk-yb-mz.fr/public/register","method":"GET"} []
[2023-02-12 17:13:47] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-02-12 17:13:47] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:13:47] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:13:47] php.CRITICAL: Uncaught Error: Class 'App\Form\Regex' not found {"exception":"[object] (Error(code: 0): Class 'App\\Form\\Regex' not found at /home/snryljie/public_html/src/Form/RegistrationFormType.php:49)"} []
[2023-02-12 17:13:47] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\ClassNotFoundError: "Attempted to load class "Regex" from namespace "App\Form". Did you forget a "use" statement for e.g. "Symfony\Polyfill\Intl\Idn\Resources\unidata\Regex" or "Symfony\Component\Validator\Constraints\Regex"?" at /home/snryljie/public_html/src/Form/RegistrationFormType.php line 49 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\ClassNotFoundError(code: 0): Attempted to load class \"Regex\" from namespace \"App\\Form\".\nDid you forget a \"use\" statement for e.g. \"Symfony\\Polyfill\\Intl\\Idn\\Resources\\unidata\\Regex\" or \"Symfony\\Component\\Validator\\Constraints\\Regex\"? at /home/snryljie/public_html/src/Form/RegistrationFormType.php:49)"} []
[2023-02-12 17:13:47] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2023-02-12 17:14:22] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Configuration\Method" annotation is deprecated since version 5.2. Use "Symfony\Component\Routing\Annotation\Route" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Configuration\\Method\" annotation is deprecated since version 5.2. Use \"Symfony\\Component\\Routing\\Annotation\\Route\" instead. at /home/snryljie/public_html/vendor/sensio/framework-extra-bundle/src/Configuration/Method.php:14)"} []
[2023-02-12 17:14:22] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Configuration\Route" annotation is deprecated since version 5.2. Use "Symfony\Component\Routing\Annotation\Route" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Configuration\\Route\" annotation is deprecated since version 5.2. Use \"Symfony\\Component\\Routing\\Annotation\\Route\" instead. at /home/snryljie/public_html/vendor/sensio/framework-extra-bundle/src/Configuration/Route.php:16)"} []
[2023-02-12 17:14:23] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"https://f2i-dev23-yk-yb-mz.fr/public/register","method":"GET"} []
[2023-02-12 17:14:23] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-02-12 17:14:23] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:14:23] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:14:23] php.CRITICAL: Uncaught Error: Class 'App\Form\Regex' not found {"exception":"[object] (Error(code: 0): Class 'App\\Form\\Regex' not found at /home/snryljie/public_html/src/Form/RegistrationFormType.php:49)"} []
[2023-02-12 17:14:23] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\ClassNotFoundError: "Attempted to load class "Regex" from namespace "App\Form". Did you forget a "use" statement for e.g. "Symfony\Polyfill\Intl\Idn\Resources\unidata\Regex" or "Symfony\Component\Validator\Constraints\Regex"?" at /home/snryljie/public_html/src/Form/RegistrationFormType.php line 49 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\ClassNotFoundError(code: 0): Attempted to load class \"Regex\" from namespace \"App\\Form\".\nDid you forget a \"use\" statement for e.g. \"Symfony\\Polyfill\\Intl\\Idn\\Resources\\unidata\\Regex\" or \"Symfony\\Component\\Validator\\Constraints\\Regex\"? at /home/snryljie/public_html/src/Form/RegistrationFormType.php:49)"} []
[2023-02-12 17:14:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2023-02-12 17:50:53] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"https://f2i-dev23-yk-yb-mz.fr/public/register","method":"POST"} []
[2023-02-12 17:50:53] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-02-12 17:50:53] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:50:53] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-02-12 17:50:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2023-02-12 17:50:54] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\UniqueConstraintViolationException: "An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 's' for key 'UNIQ_8D93D649F85E0677'" at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 62 {"exception":"[object] (Doctrine\\DBAL\\Exception\\UniqueConstraintViolationException(code: 1062): An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 's' for key 'UNIQ_8D93D649F85E0677' at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:62, Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1062): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 's' for key 'UNIQ_8D93D649F85E0677' at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 's' for key 'UNIQ_8D93D649F85E0677' at /home/snryljie/public_html/vendor/doctrine/dbal/src/Driver/PDO/Statement.php:134)"} []
[2023-03-24 12:55:38] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_route":"homepage","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2023-03-24 12:55:38] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-24 12:55:38] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 12:55:38] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 12:55:39] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-24 12:55:39] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Could not convert database value to 'array' as an error was triggered by the unserialization: 'unserialize(): Error at offset 29 of 33 bytes'")." at C:\wamp64\www\Atypik Vd\templates\base.html.twig line 59 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Could not convert database value to 'array' as an error was triggered by the unserialization: 'unserialize(): Error at offset 29 of 33 bytes'\"). at C:\\wamp64\\www\\Atypik Vd\\templates\\base.html.twig:59, Doctrine\\DBAL\\Types\\ConversionException(code: 0): Could not convert database value to 'array' as an error was triggered by the unserialization: 'unserialize(): Error at offset 29 of 33 bytes' at C:\\wamp64\\www\\Atypik Vd\\vendor\\doctrine\\dbal\\src\\Types\\ConversionException.php:117)"} []
[2023-03-24 14:02:28] request.INFO: Matched route "habitat_Add". {"route":"habitat_Add","route_parameters":{"_route":"habitat_Add","_controller":"App\\Controller\\DefaultController::addAction"},"request_uri":"http://127.0.0.1:8000/AddHabitat","method":"GET"} []
[2023-03-24 14:02:28] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-24 14:02:28] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:02:28] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:02:28] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-24 14:02:28] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"ay"} []
[2023-03-24 14:02:28] security.DEBUG: Access denied, the user is neither anonymous, nor remember-me. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:02:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: "Access Denied." at C:\wamp64\www\Atypik Vd\vendor\symfony\security-http\Firewall\ExceptionListener.php line 137 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException(code: 0): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\ExceptionListener.php:137, Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:02:28] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-24 14:06:22] request.INFO: Matched route "habitat_Add". {"route":"habitat_Add","route_parameters":{"_route":"habitat_Add","_controller":"App\\Controller\\DefaultController::addAction"},"request_uri":"http://127.0.0.1:8000/AddHabitat","method":"GET"} []
[2023-03-24 14:06:22] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-24 14:06:22] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:06:22] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:06:22] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-24 14:06:22] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"ay"} []
[2023-03-24 14:06:22] security.DEBUG: Access denied, the user is neither anonymous, nor remember-me. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:06:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: "Access Denied." at C:\wamp64\www\Atypik Vd\vendor\symfony\security-http\Firewall\ExceptionListener.php line 137 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException(code: 0): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\ExceptionListener.php:137, Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:06:22] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-24 14:07:12] request.INFO: Matched route "habitat_Add". {"route":"habitat_Add","route_parameters":{"_route":"habitat_Add","_controller":"App\\Controller\\DefaultController::addAction"},"request_uri":"http://127.0.0.1:8000/AddHabitat","method":"GET"} []
[2023-03-24 14:07:12] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-24 14:07:12] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:07:12] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:07:12] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-24 14:07:12] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"ay"} []
[2023-03-24 14:07:12] security.DEBUG: Access denied, the user is neither anonymous, nor remember-me. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:07:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: "Access Denied." at C:\wamp64\www\Atypik Vd\vendor\symfony\security-http\Firewall\ExceptionListener.php line 137 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\AccessDeniedHttpException(code: 0): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\ExceptionListener.php:137, Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\security-http\\Firewall\\AccessListener.php:91)"} []
[2023-03-24 14:07:12] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-24 14:36:37] request.INFO: Matched route "partenaire". {"route":"partenaire","route_parameters":{"_route":"partenaire","_controller":"App\\Controller\\DefaultController::partenaireAction"},"request_uri":"http://127.0.0.1:8000/devenir_partenaire","method":"GET"} []
[2023-03-24 14:36:37] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-24 14:36:37] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:36:37] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-24 14:36:37] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-24 14:36:37] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"ay"} []
[2023-03-24 14:36:37] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "" as such route does not exist.")." at C:\wamp64\www\Atypik Vd\templates\default\partenaire.html.twig line 16 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"\" as such route does not exist.\"). at C:\\wamp64\\www\\Atypik Vd\\templates\\default\\partenaire.html.twig:16, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"\" as such route does not exist. at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\routing\\Generator\\CompiledUrlGenerator.php:50)"} []
[2023-03-24 14:36:37] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-24 14:53:28] console.CRITICAL: Error thrown while running command ""make:entity"". Message: "There are no commands defined in the "make" namespace.  You may be looking for a command provided by the "MakerBundle" which is currently not installed. Try running "composer require symfony/maker-bundle --dev"." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): There are no commands defined in the \"make\" namespace.\n\nYou may be looking for a command provided by the \"MakerBundle\" which is currently not installed. Try running \"composer require symfony/maker-bundle --dev\". at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\framework-bundle\\EventListener\\SuggestMissingPackageSubscriber.php:74)","command":"\"make:entity\"","message":"There are no commands defined in the \"make\" namespace.\n\nYou may be looking for a command provided by the \"MakerBundle\" which is currently not installed. Try running \"composer require symfony/maker-bundle --dev\"."} []
[2023-03-24 14:53:55] console.CRITICAL: Error thrown while running command ""make:entity"". Message: "There are no commands defined in the "make" namespace.  You may be looking for a command provided by the "MakerBundle" which is currently not installed. Try running "composer require symfony/maker-bundle --dev"." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): There are no commands defined in the \"make\" namespace.\n\nYou may be looking for a command provided by the \"MakerBundle\" which is currently not installed. Try running \"composer require symfony/maker-bundle --dev\". at C:\\wamp64\\www\\Atypik Vd\\vendor\\symfony\\framework-bundle\\EventListener\\SuggestMissingPackageSubscriber.php:74)","command":"\"make:entity\"","message":"There are no commands defined in the \"make\" namespace.\n\nYou may be looking for a command provided by the \"MakerBundle\" which is currently not installed. Try running \"composer require symfony/maker-bundle --dev\"."} []
[2023-03-26 00:42:25] request.INFO: Matched route "users". {"route":"users","route_parameters":{"_route":"users","_controller":"App\\Controller\\DefaultController::usersAction"},"request_uri":"http://127.0.0.1:8000/users","method":"GET"} []
[2023-03-26 00:42:25] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2023-03-26 00:42:25] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-26 00:42:25] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LogInAuthenticator"} []
[2023-03-26 00:42:25] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken"} []
[2023-03-26 00:42:25] security.DEBUG: User was reloaded from a user provider. {"provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider","username":"yacein"} []
[2023-03-26 00:42:25] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Types\ConversionException: "Could not convert database value to 'array' as an error was triggered by the unserialization: 'unserialize(): Error at offset 29 of 33 bytes'" at C:\wamp64\www\Atypik Vd\vendor\doctrine\dbal\src\Types\ConversionException.php line 117 {"exception":"[object] (Doctrine\\DBAL\\Types\\ConversionException(code: 0): Could not convert database value to 'array' as an error was triggered by the unserialization: 'unserialize(): Error at offset 29 of 33 bytes' at C:\\wamp64\\www\\Atypik Vd\\vendor\\doctrine\\dbal\\src\\Types\\ConversionException.php:117)"} []
[2023-03-26 00:42:26] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
